plugins/inventory/proxmox.py

```diff
@@ -226,7 +230,13 @@ def _get_vm_config(self, node, vmid, vmtype, name):
                 if config == 'rootfs' or config.startswith(('virtio', 'sata', 'ide', 'scsi')):
                     value = ('disk_image=' + value)
 
-                if not (isinstance(value, int) or ',' not in value):
+                # Additional field containing parsed tags as list
+                if config == 'tags':
+                    parsed_key = self.to_safe('%s%s' % (key, "_parsed"))
+                    parsed_value = [tag.strip() for tag in value.split(",")]
+                    self.inventory.set_variable(name, parsed_key, parsed_value)
+
+                if not (isinstance(value, int) or ',' not in value or config in plaintext_configs):
```

Comment:
This changes behavior for `tags`. Is there a reason we cannot leave it as before?
```suggestion
                if not (isinstance(value, int) or ',' not in value):
```
If someone had `a=b, c=d` as their `tags` string, it would have resulted in a dictionary `{a=b, c=d}` before, but with your change it should result in the string `a=b, c=d`. Or is it clear from #1989 (sorry haven't read it in detail) that you cannot have `=` in the string anyway?

Reply:
Yes, so the previous behavior would always exclude tags strings containing commas because Proxmox will never allow a tag to be set containing '='.

Tags must be of the pattern ^[a-z0-9_][a-z0-9_\-\+\.]*$ and separated by ',' or ';' when in a list.